### ISSUE TITLE
Wire MethodReader fails to clear List before populating

### ISSUE THREAD

**Issue by alamar** [OpenHFT/Chronicle-Wire#894]
Please see the PR - if there is a fix please backport to x.24

**Comment by alamar** [comment]
```
    // Definition of a DTO with a list-containing field.
    public static class ListContainingDto {
        Object list;
    }
```
(see the PR)
Changing type to List will work around the issue. Adding `extends AbstractMarshallableCfg` will also would.

**Comment by tgd** [comment]
Thank you for the detailed reproducer Ilya. This is now scheduled, I will send an update soon.

**Comment by hft-team-city** [comment]
Released in [Chronicle-Wire-2.26ea1](https://github.com/OpenHFT/Chronicle-Wire/releases/chronicle-wire-2.26ea1), [BOM-2.26ea10](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.26ea10)

**Comment by hft-team-city** [comment]
Released in [Chronicle-Wire-2.25.21](https://github.com/OpenHFT/Chronicle-Wire/releases/chronicle-wire-2.25.21), [BOM-2.25.75](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.25.75)
